e2e/remix/tests/nx-remix.test.ts

```diff
@@ -0,0 +1,175 @@
+import {
+  cleanupProject,
+  killPorts,
+  newProject,
+  runCLI,
+  checkFilesExist,
+  readJson,
+  uniq,
+  updateFile,
+  runCommandAsync,
+} from '@nx/e2e/utils';
+
+describe('remix e2e', () => {
+  let proj: string;
+
+  beforeAll(() => {
+    proj = newProject();
```

Comment:
You probably would like to follow the convention that @FrozenPandaz recommended on adding just the packages we need for those tests.